src/Framework/MockObject/MockBuilder.php

```diff
@@ -178,6 +178,31 @@ public function setMethods(array $methods = null): self
         return $this;
     }
 
+    /**
+     * Specifies the subset of methods to mock, requiring each to exist in the class
+     */
+    public function setRealMethods(array $methods = null): self
```

Comment:
What do you think about removing the `null` default value?

Reply:
I think either way could make sense. The only reason I had defaulted to null is to keep it 100% backwards compatible with setMethods. When testing this is was a little beneficial in a large codebase I work on to just do a big search and replace and see the tests light up and not have to worry about compatibility, but let me know what you think, either way works.

Reply:
I understand that the method signature is consistent with that of `setMethods()`, but my questions here are: 

- What does `null` mean in this context? 
- Why would anyone want to invoke this method with`null`, or without any argument at all?

Reply:
I think you have good points, I appreciate that feedback. It does seem to not really be useful/make sense, so might as well remove. Updated and removed!